src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -119,12 +120,18 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings)
             if (packageSources.Count < 1)
             {
                 // Add enabled sources
-                var enabledSources = packageSourceProvider.Value
-                        .LoadPackageSources()
-                        .Where(source => source.IsEnabled)
-                        .Select(source => source.Source)
-                        .Distinct(StringComparer.Ordinal)
-                        .ToList();
+                foreach (PackageSource source in packageSourceProvider.Value.LoadPackageSources())
```

Comment:
I might change that since every other variable in here uses "var", but really the type helps here so that code reviewers know what "source" is. With "var" there would be no way to know except guessing.

---
In reply to: [62935456](https://github.com/NuGet/NuGet.Client/pull/586#discussion_r62935456) [](ancestors = 62935456)